sdcm/cluster.py

```diff
@@ -2551,8 +2551,8 @@ def run_nodetool(self, sub_cmd, args="", options="", timeout=None,
             except Exception as details:  # pylint: disable=broad-except
                 if isinstance(details, RetryableNetworkException):
                     details = details.original
-                if details.__class__.__name__.endswith("CommandTimedOut"):
-                    self.generate_coredump_file()
+                if coredump_on_timeout and details.__class__.__name__.endswith("CommandTimedOut"):
+                    self.generate_coredump_file(node=self.name)
```

Comment:
here you should add a message, to override the default message in the `GeneratingCoreDumpEvent` class...
it should be something like, `sending SIGQUIT because command reached timeout` to have much more detailed print before we create these coredumps on timeout